src/controllers/userController.js

```diff
@@ -0,0 +1,132 @@
+import { Router } from 'express';
+import moment from 'moment';
+import crypto from 'crypto';
+import Validation from '../validation';
+import { sendResetMail, sendSignupMail } from '../services/sendMail';
+import { errorResponse, successResponse } from '../utils/response';
+import Hash from '../utils/hash';
+import models from '../models';
+
+const { User, Login, Reset } = models;
+
+const forgotPassword = (req, res) => {
+  const { errors, isValid } = Validation.validateEmail(req.body);
+
+  // Check validation
+  if (!isValid) {
+    return errorResponse(res, 400, errors);
+  }
+
+  const { email } = req.body;
+
+  // Find user by email
+  User.findOne({ where: { email } }).then((user) => {
```

Comment:
I thought we discussed using `async/await` and `try/catch`.

Reply:
Sorry i failed to mention that at the point of refactoring to async/await, i found one use-case of .then that serves a purpose in the application logic. i.e. it provides closure for my variables and allows their life-cycle to be within the enclosing block of which they are defined. I could not do this with an async/await. Is this good enough reason for a consideration?